docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,834 @@
+---
+title: .NET Aspire 9.0 (Release Candidate 1)
+description: Learn what's new with .NET Aspire 9.0 (Release Candidate 1).
+ms.date: 10/15/2024
+---
+
+# .NET Aspire 9.0 (Release Candidate 1)
+
+.NET Aspire 9.0 Release Candidate 1 (RC1) is the next major release of .NET Aspire; it supports _both_ .NET 8 Long Term Support (LTS) and .NET 9 Standard Term Support. .NET Aspire 9 RC1 addresses some of the most highly requested features and pain points from the community. For more information on .NET version support, see the official [.NET support policy](https://dotnet.microsoft.com/platform/support/policy).
+
+## Upgrade to .NET Aspire 9 RC1
+
+Instructions on how to upgrade are detailed in the following:
+
+- [Install .NET Aspire 9 RC1 templates](#templates)
+- [Upgrade an existing project manually](#upgrade-an-existing-project-to-net-aspire-9-rc1-manually)
+- [Upgrade an existing project using the Upgrade Assistant](#upgrade-an-existing-project-to-net-aspire-9-rc1-using-the-upgrade-assistant)
+
+### Acquisition
+
+.NET Aspire 9 RC1 makes it simpler to configure your environment to develop .NET Aspire applications. You no longer need a .NET workload. Instead, add the [📦 `Aspire.AppHost.Sdk`](https://nuget.org/packages/Aspire.AppHost.Sdk) MSBuild SDK to your project and it will be downloaded via NuGet automatically.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Sdk Name="Aspire.AppHost.Sdk" Version="9.0.0-rc.1.24511.1" />
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsAspireHost>true</IsAspireHost>
+    <UserSecretsId>0afc20a6-cd99-4bf7-aae1-1359b0d45189</UserSecretsId>
```

Comment:
Are we concerned people will copy this user secrets ID into this projects?

Reply:
Shouldn't matter AFAIK. This is just a GUID to identify one in your applocal folder so that they can be matched. In fact, we have one burned in on our templates too: https://github.com/dotnet/aspire/blob/f70012e4b8f203f2357a52e8fecb4a78ae2a87b9/src/Aspire.ProjectTemplates/templates/aspire-starter/Aspire-StarterApplication.1.AppHost/Aspire-StarterApplication.1.AppHost.csproj#L11